external-crates/move/crates/move-compiler/src/typing/visitor.rs

```diff
@@ -77,6 +80,31 @@ pub trait TypingVisitorContext {
         }
     }
 
+    // TODO struct and type visiting
```

Comment:
Is there a connection between this addition and the main goal of this PR (largely for my own information)?

Reply:
Its more that I added constants and noticed we didn't have a type visitor, so largely unrelated